core/src/subsystems/tank_drive.cpp

```diff
@@ -152,13 +165,23 @@ bool TankDrive::turn_degrees(double degrees, double percent_speed)
   return false;
 }
 
+bool TankDrive::turn_degrees(double degrees, double max_speed)
```

Comment:
Would like to see a function-level comment here too, doesn't need to restate what turn_degrees does but should explain why it's separate from the other one